index.js

```diff
@@ -0,0 +1,27 @@
+const validatePassword = (password) =>{
+  const lowerCaseTest = hasLowerCase(password)
+  const upperCaseTest = hasUpperCase(password)
+  const numericValueTest = hasANumber(password)
+  const specialCharacterTest = hasSpecialCharcter(password)
+
+  if (password.length > 8 && lowerCaseTest && upperCaseTest && numericValueTest && specialCharacterTest){
+    return true 
+  }
+  else {
+    return false 
+  }
```

Comment:
Since you are returning true when the condition is true you don't need the if

```JavaScript
const validatePassword = (password) => {
  const lowerCaseTest = hasLowerCase(password)
  const upperCaseTest = hasUpperCase(password)
  const numericValueTest = hasANumber(password)
  const specialCharacterTest = hasSpecialCharcter(password)

  return password.length > 8 && lowerCaseTest && upperCaseTest && numericValueTest && specialCharacterTest
}
```